src/website/app/demos/Tooltip/examples/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import basic from './basic';
+import controlled from './controlled';
+import importSyntax from './importSyntax';
+import placement from './placement';
+import prose from './prose';
+
+export default [importSyntax, basic, prose, placement, controlled];
```

Comment:
We may need more examples to illustrate other aspects like portal, behavior in scrolling container, etc - like Popover and Dropdown.  Just because we document them there does not mean that the user will make the connection to Tooltip.  Further, keep in mind that the examples are not just for users but they are also for snapshot testing.

Reply:
Added overflow, scrolling, and portal examples.